Libraries/Components/Button.js

```diff
@@ -25,72 +25,75 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
 
 type ButtonProps = $ReadOnly<{|
   /**
-   * Text to display inside the button
+   * Text to display inside the button. On Android the given title will be converted to the uppercased form.
    */
   title: string,
 
   /**
-   * Handler to be called when the user taps the button
+   * Handler to be called when the user taps the button. The first function argument is an event in form of [PressEvent](pressevent).
    */
   onPress: (event?: PressEvent) => mixed,
 
   /**
-   * If true, doesn't play system sound on touch (Android Only)
+   * If `true`, doesn't play system sound on touch.
+   *
+   * @platform android
    **/
   touchSoundDisabled?: ?boolean,
 
   /**
-   * Color of the text (iOS), or background color of the button (Android)
+   * Color of the text (iOS), or background color of the button (Android).
    */
   color?: ?ColorValue,
 
   /**
-   * TV preferred focus (see documentation for the View component).
+   * TV preferred focus.
+   * @platform tv
    */
   hasTVPreferredFocus?: ?boolean,
 
   /**
-   * TV next focus down (see documentation for the View component).
+   * Designates the next view to receive focus when the user navigates down. See the [Android documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusDown).
    *
-   * @platform android
+   * @platform android, tv
```

Comment:
### My approach
I've used comma separated strings to specify multiple platforms. It can be easily extracted using the an automated script. Though I'm open to suggestions.

### Alternatives
- Using a list of string 
- Using multiple `@platform` tags

Reply:
I like this!

Reply:
This makes sense 👍🏻